website/docs/r/worklink_website_certificate_authority_association.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "aws"
+page_title: "AWS: aws_worklink_website_certificate_authority_association"
+sidebar_current: "docs-aws-resource-worklink-website-certificate-authority-association"
+description: |-
+  Provides a AWS WorkLink Website Certificate Authority Association resource.
+---
+
+# aws_worklink_website_certificate_authority_association
+
+## Example Usage
+
+```hcl
+resource "aws_worklink_fleet" "example" {
+  name  = "terraform-example"
+}
+
+resource "aws_worklink_website_certificate_authority_association" "test" {
+	fleet_arn	= "${aws_worklink_fleet.test.arn}"
+	certificate = "${file("test-fixtures/worklink-website-certificate-authority-association.pem")}"
```

Comment:
Nit: We may want to remove the test-fixtures part of this for the example to reduce any confusion 👍 

```
	certificate = "${file("certificate.pem")}"
```

Reply:
fixed